sections/futurework/futurework-chapter.tex

```diff
@@ -1,2 +1,22 @@
 \section{Future work}\label{sec:futurework}
-Future work, proposals for the masters thesis next semester.
\ No newline at end of file
+In the following section we will define potential future work based on the information acquired.
+In the discussion, we highlighted multiple possible directions for future work.
+
+\subsection{Side-information to improve density}
+The \textit{IS-UserBased} methods shows potential for getting good results, but due to its item splitting approach, it creates a greater level of sparsity than already exists in the datasets.
+For future work on this method, adding side-information to items or users may help alleviate this sparsity and reconnect the fictional items it creates.
+Adding this side-information to other methods such as \textit{NGCF} or \textit{LightGCN} may also be interesting to look into.
+
+\subsection{Adding context to existing methods}
+It could be interesting to look at ways to add context to \textit{LightGCN} or \textit{NGCF}.
+One way of doing this could be by employing methods to represent context in a format usable by the initial embeddings for these methods, and investigating results on the contextual datasets.
```

Comment:
måske nævn noget med at ændre deres convolutioon operationer til at tae hæjde for det